content/ideas/hackage-badges.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Build-integration and Badges for Hackage
+---
+
+The hackage docbuilder currently only gives a pass/fail with generated documentation or failure logs. Ideally we should be able to infer and present a lot more interesting data about packages to encourage package maintainers. Existence of test-suites and the extent of their coverage, success of builds with different versions, existence of benchmark suites, even extent of documentation can all be recognized with badges or shields.
+
+This work involves extending the existing docbuilder to run more detailed builds and report more detailed data, as well as extending the Hackage UI to better display data both within cabal metadata and also as generated by the builder.
+
+Additionally, it would be good to rearchitect the builder so that it doesn't store its "unbuildable" set locally, but instead is locally stateless and driven by polling the hackage server for instructions -- this allows better scale-out and parallelization of builders, as well as distribution of work.
```

Comment:
I take it from "it would be good" that this is an optional "stretch goal" rather than a requirement for the completion of this task? I think it could be made clearer that that's the case.